build/app_template_files/__NativeSwiftTemplateAppName__/__NativeSwiftTemplateAppName__/SalesforceSDKAssets.xcassets/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Shouldn't we have one copy of SalesforceSDKAssets.xcassets in the repo, and links to it in each template apps? (The way it used to be done with the Settings.bundle).